processing/src/test/java/org/apache/druid/segment/filter/ExpressionFilterTest.java

```diff
@@ -147,6 +147,7 @@ public void testOneMultiValuedStringColumn()
     }
     assertFilterMatchesSkipVectorize(edf("dim4 == '1'"), ImmutableList.of("0"));
     assertFilterMatchesSkipVectorize(edf("dim4 == '3'"), ImmutableList.of("3"));
+    assertFilterMatchesSkipVectorize(edf("dim4 == '4'"), ImmutableList.of("4", "5"));
```

Comment:
Why'd this change?

Reply:
I .. don't really remember, I'm going to go with the reason was for fun, to add another test that actually has multiple matches in the case where sql compatible null handling is enabled 😜